config-model/src/main/java/com/yahoo/vespa/model/container/component/BertEmbedder.java

```diff
@@ -16,56 +18,45 @@
  */
 public class BertEmbedder extends TypedComponent implements BertBaseEmbedderConfig.Producer {
 
-    private final ModelReference modelRef;
-    private final ModelReference vocabRef;
-    private final Integer maxTokens;
-    private final String transformerInputIds;
-    private final String transformerAttentionMask;
-    private final String transformerTokenTypeIds;
-    private final String transformerOutput;
-    private final Integer tranformerStartSequenceToken;
-    private final Integer transformerEndSequenceToken;
-    private final String poolingStrategy;
-    private final String onnxExecutionMode;
-    private final Integer onnxInteropThreads;
-    private final Integer onnxIntraopThreads;
-    private final Integer onnxGpuDevice;
```

Comment:
Only the parameters `onnxExecutionMode`, `onnxInteropThreads`, `onnxIntraopThreads` and `onnxGpuDevice` should be included in `OnnxModelOptions`. The other parameters to not control how to the Onnx model is loaded, only its input/output.

Reply:
Fixed